src/org/omegat/core/team2/impl/GITRemoteRepository2.java

```diff
@@ -201,15 +199,15 @@ protected String getCurrentVersion() throws IOException {
     public void switchToVersion(String version) throws Exception {
         try (Git git = new Git(repository)) {
             if (version == null) {
-                version = REMOTE_BRANCH;
+                version = REMOTE + "/" + getDefaultBranchName(repositoryURL);
```

Comment:
It's just a matter of style, but you can also do
```suggestion
                version = String.join("/", REMOTE, getDefaultBranchName(repositoryURL));
```